Documentation/supermuc.rst

```diff
@@ -116,17 +116,17 @@ set compiler options:
 
 ::
 
-  $ export FC=mpif90
-  $ export CXX=mpiCC
-  $ export CC=mpicc
+  $ export FC=mpiifort
```

Comment:
are you sure you need to change that?

Reply:
Anyway, this scons procedure is already obsolete, as we now use cmake (but I know that there is no documentation yet).

Reply:
Now there is a documentation. I suggest you drop the proposed changes on this file.